tests/tests.js

```diff
@@ -821,16 +969,15 @@ QUnit.test("onAuthStateChanged test", function(assert) {
 // And pretty much all of the same functions as localStorage (createCalendar, etc)
 
 QUnit.module( "CheckIt tests for clickHandlers", {
-  before: function() {
-    // prepare something once for all tests
-  },
   beforeEach: function() {
     // prepare something before each test
   },
   afterEach: function() {
     // clean up after each test
-  },
-  after: function() {
-    // clean up once after all tests are done
   }
 });
+
+
+// Have to have the same html in test html as my app html so that I can
```

Comment:
Testing is good, but you should pace yourself. Tests have to be maintained too. If you are still making UI changes then you're going to be breaking a lot of UI tests.

UI testing is not that common in projects like this because the effort required is pretty significant, but the benefit is marginal. If the UI breaks, it's usually pretty clear that things aren't working.